### PR TITLE
Update data.json

### PR DESCRIPTION
**Pull request by idamezhim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,5 +26,5 @@ Any questions let us know!
...
...sercontent.com/73097560/103608126-91b05900-4f55-11eb-84af-d30f20d9332c.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/215">(:100: give your vote)</a></p></td><td align="center"><p><a href="https://github.com/manzumrahman">Md. Manzum Rahman</a></p><img src="https://user-images.githubusercontent.com/63909405/103669582-baab0b00-4fa2-11eb-9e79-d21a269096a1.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/220">(:100: give your vote)</a></p></td></tr><tr><td align="center"><p><a href="https://github.com/TawfikYasser">Tawfik Yasser</a></p><img src="https://user-images.githubusercontent.com/54971231/103836002-6bdb9f00-5090-11eb-9501-dc21ccbf3607.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/227">(:100: give your vote)</a></p></td><td align="center"><p><a href="https://github.com/Samridhi-98">Samriddhi</a></p><img src="https://user-images.githubusercontent.com/54466041/104162252-40530e80-541a-11eb-967d-6b6f4050371c.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/235">(:100: give your vote)</a></p></td><td></td></tr></table>
```
Comment (AllanRegush) [inline README.md:29]: Please revert this change. The Action will create the Table in the README for you.

File: data.json [hunk 2]
```diff
@@ -437,12 +437,6 @@
     "imageUrl": "https://user-images.githubusercontent.com/54971231/103836002-6bdb9f00-5090-11eb-9501-dc21ccbf3607.png",
     "issueNumber": 227
   },
-  {
```
Comment (AllanRegush) [inline data.json:440]: I'm not sure why this person's entry got removed.

File: data.json [hunk 3]
```diff
@@ -454,5 +448,11 @@
...
     "issueNumber": 237
+  },
+  {
+    "name": "Douglas Franklin",
+    "githubUsername": "idamezhim",
+    "imageUrl": "https://user-images.githubusercontent.com/70078201/104479251-7f709380-55c3-11eb-9b28-4cd3bf56f7df.png",
+    "issueNumber": 241
   }
-]
+]
```
Comment (AllanRegush) [inline data.json:458]: Please add a new line to the end of the file.

**Review by AllanRegush — COMMENTED** [review]
Thanks for your PR 👍. Just a few adjustments.

File: README.md (hunk 1 above)
Comment (idamezhim) [reply README.md:29]: Thanks Allan, I just did

File: data.json (hunk 2 above)
Comment (idamezhim) [reply data.json:440]: what should I do about this?

File: data.json (hunk 3 above)
Comment (idamezhim) [reply data.json:459]: Thanks, Allan, I just added that now

File: README.md (hunk 1 above)
Comment (AllanRegush) [reply README.md:29]: Still looks like there is changes here.

File: data.json (hunk 3 above)
Comment (AllanRegush) [reply data.json:459]: Looks like the same issue. Please add the new line after the `]`

File: data.json (hunk 2 above)
Comment (AllanRegush) [reply data.json:440]: You could paste the this entry back in.

**Review by AllanRegush — COMMENTED** [review]
Just a few more tweaks. 🤓

File: README.md (hunk 1 above)
Comment (idamezhim) [reply README.md:29]: I think i got it now

File: data.json (hunk 3 above)
Comment (idamezhim) [reply data.json:459]: I just resolved, Thanks Allan

File: data.json (hunk 2 above)
Comment (idamezhim) [reply data.json:440]: I have Pasted it, Thanks Allan

**Review by idamezhim — COMMENTED** [review]
Please review for any errors again, Thanks Allan

File: data.json [hunk 4]
```diff
@@ -467,4 +473,5 @@
     "imageUrl": "https://user-images.githubusercontent.com/73777251/104731775-360a7a80-5762-11eb-8a86-b50d98b58b01.png",
     "issueNumber": 247
   }
+
```
Comment (AllanRegush) [inline data.json:476]: I think we should remove this space 🤓

File: data.json [hunk 5]
```diff
@@ -455,6 +455,12 @@
     "imageUrl": "https://user-images.githubusercontent.com/22374829/104178614-a8acea80-5430-11eb-8ee8-1a7bb89248b2.png",
     "issueNumber": 237
   },
+  {
```
Comment (AllanRegush) [inline data.json:458]: Please Append to the bottom 😄

File: README.md [hunk 6]
```diff
@@ -26,5 +26,7 @@ Any questions let us know!
...
...rc="https://user-images.githubusercontent.com/63909405/103669582-baab0b00-4fa2-11eb-9e79-d21a269096a1.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/220">(:100: give your vote)</a></p></td></tr><tr><td align="center"><p><a href="https://github.com/TawfikYasser">Tawfik Yasser</a></p><img src="https://user-images.githubusercontent.com/54971231/103836002-6bdb9f00-5090-11eb-9501-dc21ccbf3607.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/227">(:100: give your vote)</a></p></td><td align="center"><p><a href="https://github.com/Samridhi-98">Samriddhi</a></p><img src="https://user-images.githubusercontent.com/54466041/104162252-40530e80-541a-11eb-967d-6b6f4050371c.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/235">(:100: give your vote)</a></p><a href="https://github.com/idamezhim">Douglas Franklin</a></p><img src="https://user-images.githubusercontent.com/70078201/104479251-7f709380-55c3-11eb-9b28-4cd3bf56f7df.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/235">(:100: give your vote)</a></p></td><td></td></tr></table>
```
Comment (AllanRegush) [inline README.md:29]: Please remove this from the Readme. 👍

**Review by AllanRegush — COMMENTED** [review]
Looking Good. Just a bit more 👍

**Review by idamezhim — COMMENTED** [review]
Thanks, Allan

I have made the necessary corrections

File: data.json (hunk 4 above)
Comment (AllanRegush) [reply data.json:476]: Just checking in. Could you remove this space on line 476?

**Review by w3bdesign — CHANGES_REQUESTED** [review]
You should only have to modify data.json, README.md should be untouched.

The Github action will automatically generate README.md.

**Comment by eddiejaoude** [comment]
@idamezhim do you need anymore help with this?

**Comment by AllanRegush** [comment]
Just Checking in @idamezhim. If you need any help let us know 🤓

**Comment by praveenscience** [comment]
Do you need any help? Let me know if you need to resolve your Merge Conflicts.

**Comment by Vyvy-vi** [comment]
Hey @idamezhim :wave:
Do you need any help with resolving the conflicts?

File: data.json (hunk 4 above)
Comment (Panquesito7) [reply data.json:476]: I also agree with this. 👍

**Comment by schmelto** [comment]
Due to long inactivity I will close this PR now. Feel free to repopen it or create a new one.
